lib/delayed_resque/meta_data.rb

```diff
@@ -0,0 +1,21 @@
+module DelayedResque
+  class MetaData
+    def self.store_meta_data(klass, args, meta = {})
+      Resque.redis.set(key(klass, args), Resque.encode(meta))
```

Comment:
should we include an expiration maybe a month? we wont slowly leak in to the set if something does go wrong. 

https://www.rubydoc.info/github/redis/redis-rb/Redis:set

Reply:
Done.